task/12-katas-3-tasks.js

```diff
@@ -27,8 +27,48 @@
  *   'FUNCTION'  => false
  *   'NULL'      => false 
  */
+
+function dfs(puzzle, searchStr, i, j, used, cur) {
+    if (cur == searchStr.length) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"